### PR TITLE
feat: enable creation of channels with query options

### PR DESCRIPTION
**Pull request by totalimmersion**
# Submit a pull request

Added **options parameter to channel.create(), for example: `channel.create("john", hide_for_creator=True)`

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: stream_chat/async_chat/channel.py [hunk 1]
```diff
@@ -35,9 +35,12 @@ async def delete_reaction(
             params={"user_id": user_id},
         )
 
-    async def create(self, user_id: str) -> StreamResponse:
```
Comment (gumuz) [inline stream_chat/async_chat/channel.py:38]: you'd need to make the same change to the sync client as well https://github.com/GetStream/stream-chat-python/blob/master/stream_chat/channel.py#L33
this split between sync/async exists in the tests too

additionally, you'd want to update the signature in the base class too https://github.com/GetStream/stream-chat-python/blob/master/stream_chat/base/client.py

Comment (totalimmersion) [reply stream_chat/async_chat/channel.py:38]: Thanks Guyon! Just updated it.

Comment (gumuz) [reply stream_chat/async_chat/channel.py:38]: I think it's still missing a test in `stream_chat/tests/async_chat
/test_channel.py`

**Review by gumuz — APPROVED** [review]
🐍

**Pull request merged**